script.js

```diff
@@ -298,3 +307,38 @@ hamburgerButton.addEventListener('click', toggleMobileMenu);
 mobileMenuLinks.forEach((menuLink) => {
   menuLink.addEventListener('click', toggleMobileMenu);
 });
+form.addEventListener('submit', (event) => {
+  event.preventDefault();
+  if (nameInput.value !== '' && messageInput.value !== '' && emailPattern.test(emailInput.value)) {
+    form.submit();
+    formErrorMessage.textContent = '';
+    emailErrorMessage.style.display = 'none';
+    nameErrorMessage.style.display = 'none';
+    messageErrorMessage.style.display = 'none';
+  } else {
+    formErrorMessage.textContent = 'the form was not sent';
+
+    if (!emailPattern.test(emailInput.value)) {
+      emailErrorMessage.textContent = 'the email input should follow this format : xyz@exe.com';
+      emailErrorMessage.style.display = 'block';
```

Comment:
- [OPTIONAL] Great work implementing so many validations. Your work is impressive. For better user experience and interaction, I would advise you to make the message more intuitive. Currently, the message doesn't explain clearly why the user's form data wasn't sent. I would advise you to explain that all characters inputted should be in _**lowercase**_ and they must follow the format `xyz@exe.com`